### PR TITLE
Update devel/valgrind

### PR DESCRIPTION
**Pull request by 1338-1**

### CODE REVIEW

File: devel/valgrind/pkg-descr [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Valgrind is a flexible program for debugging and profiling Linux executables.
```
Comment (daftaupe) [inline devel/valgrind/pkg-descr:1]: Should this description be adapted to DragonFly maybe ?

Comment (1338-1) [reply devel/valgrind/pkg-descr]: Changed.

**Review by daftaupe — APPROVED** [review]

**Comment by tuxillo** [comment]
@1338-1 is the work in your repo finished? If so, would it be possible that you provided branches for each version there?
For example, instead of having a 'dragonfly' branch, have a 'dragonfly-3.15' branch so that we can easily diff against it for updates.

**Comment by 1338-1** [comment]
> @1338-1 is the work in your repo finished? If so, would it be possible that you provided branches for each version there?
> For example, instead of having a 'dragonfly' branch, have a 'dragonfly-3.15' branch so that we can easily diff against it for updates.

Isn't that the point of tags? I made a v3.15.0 release

**Comment by tuxillo** [comment]
> Isn't that the point of tags? I made a v3.15.0 release

Yes, tags can be used too, whatever you prefer. For this type of work I use branches because if you later find something you need to fix for a specific version, you have to bother recreating the tag.

**Comment by 1338-1** [comment]
> > Isn't that the point of tags? I made a v3.15.0 release
> 
> Yes, tags can be used too, whatever you prefer. For this type of work I use branches because if you later find something you need to fix for a specific version, you have to bother recreating the tag.

I'd prefer to keep the tags, I'm more used them so I guess this shouldn't be a problem.

**Comment by tuxillo** [comment]
Committed in: https://github.com/DragonFlyBSD/DPorts/commit/36c2cc9308fc10bb309203d0a4d09c655d9e2bc0

**Pull request closed without merging**